server.js

```diff
@@ -3,16 +3,16 @@
 require('core-js/fn/object/assign');
 const webpack = require('webpack');
 const WebpackDevServer = require('webpack-dev-server');
-const config = require('./webpack.config');
+const config = require('./webpack.dev');
 const open = require('open');
 
 let server = new WebpackDevServer(webpack(config), config.devServer);
 
-server.listen(config.port, '0.0.0.0', (err) => {
```

Comment:
Is there a particular reason why you hardcoded the port?

Reply:
Nope! I don't remember why I did that, and there's no reason it can't come from the config. I'll update that.

Reply:
Thanks! 👍

Reply:
Realized this file isn't actually used, removed it.